tcms_tap_plugin/__init__.py

```diff
@@ -14,8 +14,25 @@ def __init__(self):
     def parse(self, tap_file, progress_cb=None):
         self.backend.configure()
 
+        before_first_test = True
+        test_case_run_id = False
+        status_id = False
+        trace_back = []
+        comment = ''
         for line in Parser().parse_file(tap_file):
-            if not isinstance(line, Result):
+            if isinstance(line, Result):
+                # Before parse result line, we need to add
+                # traceback as comment to the previous result
+                if test_case_run_id and status_id and trace_back:
+                    comment += '\n' + '\n'.join(trace_back)
+                    self.backend.update_test_case_run(
+                        test_case_run_id, status_id, comment)
```

Comment:
I'm not liking that we update the status again, and need to carry around the `status_id` variable.

I am going to expose a method  in the helper backend which lets you add comments.